web/ui/react-app/src/pages/graph/Graph.tsx

```diff
@@ -120,19 +184,32 @@ class Graph extends PureComponent<GraphProps, GraphState> {
 
   handleResize = () => {
     if (isPresent(this.$chart)) {
-      this.plot(this.$chart.getData() as GraphSeries[]);
+      this.plot(this.$chart.getData() as (GraphSeries | GraphExemplar)[]);
     }
   };
 
   render() {
-    const { chartData } = this.state;
+    const { chartData, selectedExemplarLabels } = this.state;
+    const selectedLabels = selectedExemplarLabels as { [key: string]: string };
     return (
       <div className="graph">
         <ReactResizeDetector handleWidth onResize={this.handleResize} skipOnMount />
         <div className="graph-chart" ref={this.chartRef} />
+        {Object.keys(selectedLabels).length > 0 ? (
+          <div className="float-right">
+            <span style={{ fontSize: '17px' }}>Selected exemplar:</span>
+            <div className="labels mt-1">
+              {Object.keys(selectedLabels).map((k, i) => (
+                <div key={i} style={{ fontSize: '15px' }}>
```

Comment:
For some reason, Tailwind wasn't applying these `font-size`s (as well as some other properties) so I had to resort to style props

Reply:
@LeviHarrison We're not using Tailwind, so that might be the reason? We do have a number of CSS / SCSS (for the theme stuff) files though which we use in addition to inline styles. So you could use e.g. https://github.com/prometheus/prometheus/blob/main/web/ui/react-app/src/themes/_shared.scss

Reply:
Style-wise I'd prefer the font sizes and styles to be similar to the legend items, and instead of floating the exemplar details to the right, how about inserting another full-width section above the legend (equally indented, etc.)? Keeps things simpler / more predictable in the face of shifting window + legend widths.